api/src/main/java/org/apache/iceberg/expressions/Expressions.java

```diff
@@ -251,7 +251,7 @@ public static Expression rewriteNot(Expression expr) {
     return ExpressionVisitors.visit(expr, RewriteNot.get());
   }
 
-  static <T> NamedReference<T> ref(String name) {
+  public static <T> NamedReference<T> ref(String name) {
```

Comment:
We are using this in `Builder`.

Reply:
It should be fine to pass this in through the factory methods that accept `Term`, so it's okay to make it public.

It should have Javadoc, though. And it would be helpful to mention that this is equivalent to using the `Expressions` methods that accept `String`: `The following are equivalent: equals("a", 5) and equals(ref("a"), 5)`

Reply:
Will add.